vnext/local-cli/config/configUtils.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ * @format
+ */
+// @ts-check
+
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+
+const xmldoc = require('xmldoc');
+
+// find VS files matching the pattern
+function findFiles(folder, filenamePattern) {
+  const files = glob.sync(path.join('**', filenamePattern), {
+    cwd: folder,
+    ignore: [
+      'node_modules/**',
+      '**/Debug/**',
+      '**/Release/**',
+      '**/Generated Files/**',
+      '**/packages/**',
```

Comment:
Blacklisting any directory or subdirectory named "packages" feels a bit broad. Could we remove or tighten this one?

Reply:
Unfortunately that's where nuget packages get restored and we don't want to search there. We could remove the path, but I'd rather leave it here and if someone happens to hit the small chance to have a conflict, they can use the override.

Reply:
I mean that we do this right now for anything named packages. E.g `foo/bar/packages`, instead of just the one we know we care about.

Reply:
We've hit this all the time even in our repo with people being to general with .gitignore files for example. If we know where a directory lives, we shouldn't blacklist every subdirectory that happens to have the same name.

Reply:
The glob is run on the working directory which is the windows folder under the particular app/ native module lib root. There are no packages folders under windows folders in our repo, nor do we put ones there for new apps. The *only* case this is a problem is if a user literally creates an RN app named "packages" and then runs our init, creating a windows folder with `packages.sln` and a `packages\packages.vcxproj`, in which case they're probably going to have a lot of problems with VS anyway. Won't fix.